src/main/java/com/robosh/ejournal/entity/admin/Admin.java

```diff
@@ -0,0 +1,51 @@
+package com.robosh.ejournal.entity.admin;
+
+import com.robosh.ejournal.entity.School;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.lang.Nullable;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+public class Admin {
+    @Id
```

Comment:
we have empty line here in all other classes

Reply:
Ok, I'll fix it now